www/js/con-author.js

```diff
@@ -132,7 +132,7 @@ define([
 		},
 
 		resettableControls: ["variable","description","value","units","equation"],
-		variableNodeControls: ["variable","value","units","kind","dynamic","root"],
+		variableNodeControls: ["variable","value","units","kind","root"],
```

Comment:
No, because, we use this list to initialize all the node fields in initialization part of node editor, and as we do not get the radio buttons by id., instead, we get them by name using dojo query, this field is  handled explicitly in the initialization part of node editor